data/bactrian.txt

```diff
@@ -34,7 +34,7 @@ mission "Bactrian Unlock [0]"
 					defer
 					
 			`	"We were wondering if you would be able to help us. We have a record of you having assisted in a number of humanitarian ventures before."`
-			`	The woman pulls out a tablet of some sort and begins reading from it. "Helping to terraform Rand, rescuing miners on a Syndicate world, transporting a few scientists conducting research on Hope." She continues on to mention a few more situations that you were involved in and finally ends on you helping Joe and Maria off of Smuggler's Den, although she doesn't mention them by name.`
+			`	The woman pulls out a tablet of some sort and begins reading from it. "Helping to terraform Rand, rescuing miners on a Syndicate world, transporting a few scientists conducting research on Hope..." She continues on to mention a few more situations that you were involved in and finally ends on you helping Joe and Maria off of Smuggler's Den, although she doesn't mention them by name.`
```

Comment:
I translated Dutch to English & was like "sure, only the "ie" becomes "ion"
just remove the on so ya get "She continues, mentioning ~" seems fine